index.html

```diff
@@ -20,8 +20,8 @@
 
     <div class="container">
 
-        <header>
-            <nav>
+        <header class = 'head'>
```

Comment:
You shouldn't had to have changed the html to get it working but as long as it's working why not 👍